dashboard/config/routes.rb

```diff
@@ -64,6 +64,8 @@ module OPS
   get 'docs/*path', to: 'curriculum_proxy#get_doc'
   get 'curriculum/*path', to: 'curriculum_proxy#get_curriculum'
 
+  post 'teacherFeedback/create', to: 'teacher_feedbacks#create'
```

Comment:
Rather than creating individual routes, this should be a [resource route](http://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default).
```ruby
# add more actions to the array as needed
resource :teacher_feedback, only: [:show, :create]
``` 

Also since this is an API controller, it should be contained in the [api/v1 namespace](https://github.com/code-dot-org/code-dot-org/blob/5aac5890c82061014971687178bf2cc325da29e9/dashboard/config/routes.rb#L578-L579).